src/sample-python/sample_python.py

```diff
@@ -0,0 +1,82 @@
+import requests
+import json
+'''
+Save subscription key in a .txt file to avoid passing subscription key in command line every run.
+Format:
+<subscription_key>
+
+Replace <subscription_key> with key provided.
+E.g
+asnk603nlk5f7
+'''
+def get_subscription_key():
+    key_filepath = "src\subscription_key.txt"
```

Comment:
I think this might not work when somebone execute this file (path should be ../ ?)